ovs/extensions/services/service.py

```diff
@@ -15,14 +15,20 @@
 """
 Service Factory module
 """
+import time
 
-from subprocess import check_output
+from subprocess import check_output, CalledProcessError
 from ovs.extensions.services.upstart import Upstart
 from ovs.extensions.services.systemd import Systemd
 from ovs.log.logHandler import LogHandler
 
 logger = LogHandler.get('extensions', name='servicemanager')
-
+try:
+    from ovs.extensions.services.fleetctl import FleetCtl
```

Comment:
It is interesting on newer Ubuntu which has systemd as well, but indeed, could be moved in the "else" part of service detection.